dataprep/eda/basic/compute.py

```diff
@@ -1139,3 +1286,31 @@ def _get_timeunit(min_time: pd.Timestamp, max_time: pd.Timestamp, dflt: int) ->
         prev_unit = unit
 
     return prev_unit
+
+
+def _format_values(key: str, value: Any) -> str:
```

Comment:
Can you please use MB if the size of the column is greater than 1000KB? Please also add a brief function description